qimgv/components/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ if(APPLE)
     #    directorymanager/watchers/linux/linuxfsevent.cpp
     #    directorymanager/watchers/linux/linuxwatcher.cpp
     #    directorymanager/watchers/linux/linuxworker.cpp)
-elseif(UNIX AND NOT (CMAKE_SYSTEM_NAME MATCHES ^FreeBSD$))
+elseif(UNIX)
```

Comment:
Instead of leaving this out, it can be supported via `devel/libinotify-kqueue`.

Reply:
Can't find libinotify-kqueue.
Maybe devel/libinotify?

Reply:
Yes, right.